cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_ctrl_repositories.go

```diff
@@ -11,3 +11,42 @@
 // limitations under the License.
 // */
 package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/kapp_controller/packages/v1alpha1"
+	log "k8s.io/klog/v2"
+)
+
+// GetPackageRepositories returns the package repositories based on the request managed by the 'kapp_controller' plugin
```

Comment:
Note: we don't necessarily need this yet. We're not yet using it in the dashboard and the API is something Greg's working on in the background. We just had it initially with the kapp-controller plugin as an example, I think.

Reply:
Yep, I know, but I didn't want to remove the existing code and, given that we are gonna work on that soon, it worths the effort on using the same from/to unstructured approach, for example.